src/main/java/com/machowina/model/ParkingZone.java

```diff
@@ -0,0 +1,55 @@
+package com.machowina.model;
+
+import java.util.Set;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+
+import org.hibernate.validator.constraints.NotBlank;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Data
+@NoArgsConstructor
+@Entity
+public class ParkingZone {
+	
+	@Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+	
+	@NotBlank
+	private String city;
+
+	@NotBlank
+	private String name;
+	
+	@OneToMany(cascade = CascadeType.ALL)
+	@JoinColumn(name = "owner_zone_id")
+	private Set<User> owners;
```

Comment:
A jak będziemy mieli tu tysiące użytkowników to potrzebujemy ich wszystkich wyciągnąć?

Reply:
Zakładam że właściciele i operatorzy strefy nie są liczeni w tysiącach, ale można by przenieść tę relację na stronę usera